package.json

```diff
@@ -39,6 +39,7 @@
     "@sentry/webpack-plugin": "^1.15.1",
     "@types/express-rate-limit": "^5.1.3",
     "aws-sdk": "^2.1101.0",
+    "axios": "^0.27.2",
```

Comment:
tech debt to address next time: the codebase currently using `cross-fetch` for making HTTP requests. axios might not be necessary

Reply:
Aha! Thanks for highlighting, I was poking around and couldn't find a HTTP request module.

Reply:
no worries we can address this next time!